### PR TITLE
Payara 1451

### PR DESCRIPTION
**Pull request by smillidge**
Support custom Qualifiers on events sent via the Clustered CDI Event Bus.

### CODE REVIEW

**Comment by smillidge** [comment]
jenkins test please

**Comment by smillidge** [comment]
OK I'll test it against your HZ fixes to see whether I need the value holder. It''l be a good test.

**Comment by smillidge** [comment]
Changed to Utility. Value Holder is still required as well but not just for Hazelcast.

File: appserver/payara-appserver-modules/payara-jsr107/src/main/java/fish/payara/cdi/jsr107/implementation/PayaraTCCLObjectInputStream.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public PayaraTCCLObjectInputStream(InputStream is) throws IOException {
 
     @Override
     protected Class<?> resolveClass(ObjectStreamClass desc) throws IOException, ClassNotFoundException {
-        return Thread.currentThread().getContextClassLoader().loadClass(desc.getName());
+        return Class.forName(desc.getName(), false, Thread.currentThread().getContextClassLoader());
```
Comment (lprimak) [inline appserver/payara-appserver-modules/payara-jsr107/src/main/java/fish/payara/cdi/jsr107/implementation/PayaraTCCLObjectInputStream.java]: Use Utility.getClassLoader() stead

**Comment by smillidge** [comment]
jenkins test please

**Comment by smillidge** [comment]
@lprimak please approve if you are happy after review.

**Comment by lprimak** [comment]
jenkins test please

**Pull request merged**